tests/unit_tests/pyanaconda_tests/test_kickstart_specification.py

```diff
@@ -406,6 +406,7 @@ class ModuleSpecificationsTestCase(unittest.TestCase):
 
     # Names of the kickstart commands and data that should be temporarily ignored.
     IGNORED_NAMES = {
+        "SshKeyData"
```

Comment:
Is there anything else needed on our side? Shouldn't we just bump the version of `SshKeyData`?

Reply:
Oh, sorry, you are right. I missed that this was released too and thought it's only in the copr nightlies. Pushed the right version now.

Reply:
I think we don't use `pykickstart` from the copr nightlies anymore, but I am not sure.

Reply:
That actually describes well the state of infrastructure these days: Nobody really knows \o/